### ISSUE TITLE
"composer require twilio/sdk" fails

### ISSUE THREAD

**Issue by SnowLeopard71** [twilio/twilio-php#565]
I tried installing using _composer_ and all I got was this unhelpful error message:
```
$ composer require twilio/sdk
Using version ^5.35 for twilio/sdk
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)

Installation failed, deleting ./composer.json.

  [ErrorException]
  "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?

require [--dev] [--prefer-source] [--prefer-dist] [--no-progress] [--no-update] [--no-scripts] [--update-no-dev] [--update-with-dependencies] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [--sort-packages] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--] [<packages>]...


```

```
$ php --version
PHP 7.3.5-1+0~20190503093827.38+stretch~1.gbp60a41b (cli) (built: May  3 2019 09:38:28) ( NTS )
```

**Comment by SnowLeopard71** [comment]
Problem is a result of the combination of _php 7.3_ and _composer 1.2.2_ on Debian Stretch (9.9).  A different Debian machine with only _php 7.0_ had no problem.
Changing the _continure_ to _break_ in _/usr/share/php/Composer/DependencyResolver/RuleSetGenerator.php_ at line 286 allows the installation process to work.

**Comment by childish-sambino** [comment]
We're working on releasing an update to the lib that is compatible with PHP 7.1+

Hopefully next week.

**Comment by childish-sambino** [comment]
Should be fixed in this [5.36.0](https://github.com/twilio/twilio-php/releases/tag/5.36.0) release.
